tests/fixtures/source-code-util/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+    // This file is only to ensure no error is thrown due to lack of config.
```

Comment:
@IanVS Acknowledged, won't change.

What I was referring to is, since JSON doesn't support comments, some folks will just add a top-level key "__comment" (or similar) to store their file comments.